code/modules/clothing/spacesuits/rig/modules/ninja.dm

```diff
@@ -205,3 +208,108 @@
 		holder.wearer.drop_from_inventory(src)
 		qdel(holder)
 	qdel(src)
+
+/obj/item/rig_module/emp_shielding
+	name = "EMP dissipation module"
+	desc = "A bewilderingly complex bundle of fiber optics and chips. Seems like it uses a good deal of power."
+	active_power_cost = 10
```

Comment:
Add a use power cost and a cooldown, so it cant be spammed on and off.
(Like its done with the stealth field)